web/packages/teleport/src/services/recordings/makeRecording.ts

```diff
@@ -119,4 +123,42 @@ function formatDuration(startDateString: string, stopDateString: string) {
   return { duration, durationText };
 }
 
+function makeDatabaseRecording({
+  time,
+  session_start,
+  session_stop,
+  user,
+  sid,
+  db_name,
+  db_protocol,
+}) {
+  const description = cfg.getPlayableDatabaseProtocols().includes(db_protocol) ? 'play' : 'non-interactive';
+  let { duration, durationText } = formatDuration(
+    session_start,
+    session_stop
+  );
+
+  // Older database session recordings won't have start/stop fields. For those
+  // recordings we set the duration to the smallest number so we can still
+  // them.
+  // As a side effect, the progress bar to not work properly, showing always as
+  // completed. Also, navigating through it won't work.
+  if (duration === 0) {
+    duration = 1;
+    durationText = '-';
+  }
+
+  return {
+    duration,
+    durationText,
+    sid,
+    createdDate: time,
+    users: user,
+    hostname: db_name,
```

Comment:
Should this be the database service name?
```suggestion
    hostname: db_service_name,
```

 Ideally, we should db_user, db_service, and db_name info some where in this table. Or do we print out db_user and db_name info when playing?

Reply:
> Ideally, we should show db_user, db_service, and db_name info some where in this table.

That's correct, I'll update it.

> Or do we print out db_user and db_name info when playing?

I'm adding this information to the session recording itself. (I'll update the other PR soon).